ci/tests/mgmtcenter/mgmt_center_test.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2015 iNuron NV
```

Comment:
General remark about this testsuite

In setup:  you take 1st mgmtcenter if there are already mgmtcenters present
If not, you create your own mgmtcenter

Then each retrieves all managementcenters if none provided via arguments and then the checks are performed, but the setup only configures max 1 mgmtcenter

==> if 2 or more mgmtcenters already exist, this testsuite will always fail